app/models/minion.rb

```diff
@@ -0,0 +1,4 @@
+# Minion represents the minions that have been registered in this application.
+class Minion < ApplicationRecord
+  default_scope { order hostname: :asc }
```

Comment:
Right. That's why I said that we can remove it if you want. I think it has more drawbacks than advantages.